pages.zh/linux/unix2mac.md

```diff
@@ -0,0 +1,12 @@
+# unix2mac
+
+> 将 Unix 样式的行尾更改为 macOS 样式.
+> 用 LF 替换 CR.
+
+- 更改文件的行尾:
+
+`unix2mac {{filename}}`
```

Comment:
Missed one here I think? `filename` here should be ok to translate :-)

Reply:
Fixed.